pypeit/tests/test_metadata.py

```diff
@@ -52,7 +52,7 @@ def test_read_combid():
 
     indx = pmd['filename'] == 'b27.fits.gz'
     assert pmd['comb_id'][indx] == [1], 'Incorrect combination group ID'
-    assert pmd['comb_id'][~indx] == [-1], 'Incorrect combination group ID'
+    assert pmd['comb_id'][np.where(~indx)[0]][0] == -1, 'Incorrect combination group ID'
```

Comment:
Why was the previous version failing?  It seems like these two methods should be identical.

Reply:
because we went from one file to many.  It was not well written initially.